scoap3/modules/tools/tools.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+from flask import current_app
+from invenio_search import current_search_client
+
+from scoap3.modules.records.util import get_first_journal, get_first_doi, get_arxiv_primary_category
+from scoap3.utils.arxiv import get_clean_arXiv_id
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_query_string(**kwargs):
+    """
+    Concatenates the non-None keyword arguments to create a query string for ElasticSearch.
+
+    :return: concatenated query string or None if not arguments were given
+    """
+    q = ['%s:%s' % (key, value) for key, value in kwargs.items() if value is not None]
+    return ' AND '.join(q) or None
+
+
+def affiliations_export(country=None, year=None):
+    """
+    Creates affiliation data filtered by country and year.
+
+    :param country: only affiliations for this country will be included. If None, all countries are included.
+    :param year: only articles *published* in this year will be included. If None, all articles are included.
+    """
+
+    size = 100  # paging size for ElasticSearch
+    search_index = current_app.config.get('SEARCH_UI_SEARCH_INDEX')
+    source_fields = [
```

Comment:
The `size` should be in config, but the `source_fields` I would say not, because that is strongly connected to the other parts of the function, but not used anywhere else.

Reply:
If we put the `source_fields` in the config we will have to create a new config variable for each tool.